File: frontend/src/types/exceptions.ts
import type { Sale } from './sale'

export interface ExceptionCenter {
  attention_count: number
  pending_documents_count: number
  document_failures_count: number
  refunds_needing_attention_count: number
  incomplete_details_count: number
  pending_documents: Sale[]
  document_failures: Sale[]
  refunds_needing_attention: Sale[]
  incomplete_details: Sale[]
}

File: backend/app/schemas/exceptions.py
from pydantic import BaseModel

from app.schemas.sale import SaleRead


class ExceptionCenterResponse(BaseModel):
    attention_count: int
    pending_documents: list[SaleRead]
    document_failures: list[SaleRead]
    refunds_needing_attention: list[SaleRead]
    incomplete_details: list[SaleRead]
